pkg/convert/json_nglocalize_test.go

```diff
@@ -48,6 +48,33 @@ func Test_FromNgLocalize(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "Message with special chars",
+			input: []byte(`
+      {
+        "locale": "fr",
+        "translations": {
+          "Hello": "Bonjour {le monde}",
```

Comment:
Perhaps it would be good to find real-world examples to supplement the test with more cases.